Sources/EpoxyBars/BarModel/AnyBarModel.swift

```diff
@@ -60,3 +61,51 @@ extension AnyBarModel: Diffable {
     return model.isDiffableItemEqual(to: otherDiffableItem)
   }
 }
+
+// MARK: CallbackContextEpoxyModeled
+
+extension AnyBarModel: CallbackContextEpoxyModeled {
+  public typealias CallbackContext = Void
+}
+
+// MARK: WillDisplayProviding
+
+extension AnyBarModel: WillDisplayProviding {}
```

Comment:
Would it be possible to write some tests for this behavior in the style of `CollectionViewSpec.swift`? Just to verify these methods are getting called correctly on an installed bar in the case of type erasure:
https://github.com/airbnb/epoxy-ios/blob/master/Tests/EpoxyTests/CollectionViewTests/CollectionViewSpec.swift#L27

Reply:
Yep, I'll write some unit tests. Just wanted to solidify direction first